requirements_bundles.txt

```diff
@@ -6,4 +6,3 @@
 # These can be removed when upgrading to Python 3.x
 importlib-metadata>=0.19  # remove when on 3.8
 importlib_resources==1.0.2  # remove when on 3.7
```

Comment:
I think his can be removed as well (and the import updated in `bin/bundle-extensions`) for the importlib.resources package.

Reply:
This is actually used by the container building the frontend which uses the default Python version shipped with it (I think Python 3.5) so this is still needed for now.

Reply:
Ah good point!